.github/workflows/integration-tests-workflow.yml

```diff
@@ -150,12 +165,12 @@ jobs:
         if: ${{ runner.os == 'Linux' }}
         run: |
           cd ./OpenSearch-Dashboards
-          if timeout 600 grep -q "bundles compiled successfully after" <(tail -n0 -f dashboard.log); then
-            echo "OpenSearch Dashboards compiled successfully."
+          if timeout 60 grep -q "http server running" <(tail -n +1 -f dashboard.log); then
+            echo "OpenSearch Dashboards started successfully."
           else
-            echo "Timeout for 600 seconds reached. OpenSearch Dashboards did not finish compiling."
+            echo "Timeout of 60 seconds reached. OpenSearch Dashboards did not start successfully."
             exit 1
-          fi
+          fi&
```

Comment:
if you put to background it will continue to next step even if `http server running` didn't appear in `dashboard.log` yet, right? and if 60 seconds passed, will `exit 1` be able to abort the workflow if it has already moved to the next step?